### PR TITLE
Dependency/aj 249 remove dependency

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

File: src/main/java/clickup/requests/DependencyRequest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * Copyright (c) 2021 JalaSoft.
+ * This software is the confidential and proprietary information of JalaSoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with JalaSoft
+ *
+ * @author Jorge Caceres
+ */
+
+
```
Comment (Raymundogs) [inline src/main/java/clickup/requests/DependencyRequest.java]: it should be only one new line.

File: src/main/java/core/api/ApiRequestBuilder.java [hunk 2]
```diff
@@ -93,6 +93,17 @@ public ApiRequestBuilder cleanParams() {
         return this;
     }
 
+    /**
+     * Removes all tquery params.
```
Comment (Raymundogs) [inline src/main/java/core/api/ApiRequestBuilder.java]: just query without the t

**Review by Raymundogs — APPROVED** [review]

File: src/main/java/clickup/requests/DependencyRequest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    private ApiRequest apiRequest;
+
+    /**
+     * Add a dependency to a task.
+     *
+     * @param id to be set on the tag
+     * @author Jorge Caceres
+     */
+    public void addDependency(final String id) {
+        JSONObject jsonBody = new JSONObject();
```
Comment (GustavoZHA) [inline src/main/java/clickup/requests/DependencyRequest.java:29]: Is not better to create or use a Dependency entity?

Comment (JorgeRCV) [reply src/main/java/clickup/requests/DependencyRequest.java:29]: The dependency only has one value, and I think an entity wont be the best option

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**